app/controllers/admin/order_statuses_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Admin
+  class OrderStatusesController < StatusesController
+    private
+
+    def resource
+      @order ||= Order.find(params[:id]) # rubocop:disable Naming/MemoizedInstanceVariableName
```

Comment:
Rubocop enforces using the same name for memoized variables and their methods. I used different names as I was  initially afraid of conflict between different controllers. But now i believe it should be fine if changed to 
```suggestion
      @resource ||= Order.find(params[:id])
```
as the variable is only available to the individual instances of controllers.

Reply:
I think @resource will be fine for all of them.
But disable rubocop if you now what you doing is also fine ;)